updatesearch/metadata.py

```diff
@@ -145,20 +189,141 @@ def pipeline_to_xml(self, article):
         if self.load_indicators is True:
             pipeline_itens.append(pipeline_xml.ReceivedCitations())
 
-        pipeline_itens.append(pipeline_xml.TearDown())
+        add = ET.Element('add')
 
-        ppl = plumber.Pipeline(*pipeline_itens)
+        if self.include_cited_references:
+            article_pipeline_results = plumber.Pipeline(*pipeline_itens).run([article])
 
-        xmls = ppl.run([article])
+            cit_xmls, citations_fk_doc = self.get_xmls_citations(article)
 
-        # Add root document
-        add = ET.Element('add')
+            for cit_xml in cit_xmls:
+                add.append(cit_xml)
+
+            # Completa pipeline de artigo com informações estrangeiras das referências citadas
+            self.add_fields_to_doc(article_pipeline_results, citations_fk_doc)
 
-        for xml in xmls:
-            add.append(xml)
+            add.append(citations_fk_doc)
+
+        else:
+            pipeline_itens.append(pipeline_xml.TearDown())
+            article_pipeline_results = plumber.Pipeline(*pipeline_itens).run([article])
+
+            for tag in article_pipeline_results:
+                add.append(tag)
```

Comment:
usar `extend(article_pipeline_results)` no lugar `for + append`

Reply:
há outros pontos que tem a mesma situação

Reply:
Ok, já verifiquei todos aqueles inseridos neste PR.